### ISSUE TITLE
[Feature Request] An ignore feature with text-replacement.

### ISSUE THREAD

**Issue by simbalion** [vector-im/riot-web#3465]
I am of the opinion that /ignore is the most important feature for a group chat app. However, a simple /ignore such as IRC has is inadequate, as it actually cripples the user, who now can't see half the conversation taking place.

What is superior and should be standard, is an ignore which replaces the blocked user's text with something else. This way you can see that the user said something, so you can follow the conversation, but without being forced to read their dumb or offensive words.

There is a weechat plugin which does this by replacing their text with pre-configured and randomly chosen strings. I use a selection of inspirational quotes, but ideally a person could use any sort of strings they desire.

**Comment by lampholder** [comment]
This would be neat - is https://weechat.org/scripts/source/curiousignore.pl.html/ the weechat plugin?

**Comment by simbalion** [comment]
Nope, here is the weechat plugin: https://weechat.org/scripts/source/shutup.py.html/

The one you linked would be suitable too, however it's not quite as cool as replacing their text with inspirational quotes, as that is also ironic. :)
